specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -426,6 +426,103 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HanaOnAzure/sapMonitors/{sapMonitorName}": {
+      "get": {
+        "tags": [
+          "HanaOnAzure"
+        ],
+        "operationId": "SapMonitors_Get",
+        "summary": "Gets properties of a SAP monitor.",
+        "description": "Gets properties of a SAP monitor for the specified subscription, resource group, and resource name.",
+        "x-ms-examples": {
+          "Get properties of a SAP monitor": {
+            "$ref": "./examples/SapMonitors_Get.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SapMonitorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SapMonitor"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
>put [](start = 7, length = 3)

Tracked resources also require implementing patch. Patch should at minimum allow you to update tags on the resource.

Reply:
done, added patch